DOC Review forms section (#254)
silverstripe/developer-docs: en/02_Developer_Guides/03_Forms/How_Tos/05_Simple_Contact_Form.md

```

[hint]
Caution: This form is prone to abuse by spammers,

> since it doesn't enforce a rate limitation, or checks for bots.

Reviewer: ```suggestion
since it doesn't enforce rate limitation and doesn't implement any checks for bots.
```

Reviewer 2: Done
